src/api/status/public/js/build-log/api.js

```diff
@@ -0,0 +1,42 @@
+function getCurrentUrl() {
+  // http://localhost:1111/pages/build.html -> http://localhost:4000/status
+  return `${location.protocol}//${location.hostname}:4000/status`;
```

Comment:
You can use `//` and not bother with the protocol, it will use the same protocol as the current URL.